tidy3d/components/viz.py

```diff
@@ -73,18 +73,15 @@ def _plot(*args, **kwargs) -> Ax:
 """ plot parameters """
 
 
-class PlotParams(Tidy3dBaseModel):
-    """Stores plotting parameters / specifications for a given model."""
+class AbstractPlotParams(Tidy3dBaseModel):
```

Comment:
@tylerflex Might want to check my changes here which I used for plotting the path integrals